pkg/kubectl/cmd/util/editor/example_test.go

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
Yeah, unfortunately, this is indeed waaaaay to complicated for an example. 

Can we have a MUCH simpler version of "example" that compiles, but doesn't run? (examples with no expected output don't actually run).